run_dtests.py

```diff
@@ -104,6 +104,13 @@ def run(self, argv):
             logging.root.setLevel(logging.DEBUG)
             logger.setLevel(logging.DEBUG)
 
+            # cause logger to go to stdout
```

Comment:
while debugging I found that the logging logic here gets ignored, so added this so debugging actually works.